tests/semantic/CMakeLists.txt

```diff
@@ -227,4 +227,8 @@ positive_test(pragma1)
 souffle_run_test_helper(TEST_NAME pragma2 FUNCTORS CATEGORY semantic)
 positive_test(rel_redundant)
 positive_test(type_as4)
+if (FALSE)
+# semantics checker of complex rule does not catch this
+# "variable only occurs once" issue.
 positive_test(complex_rule)
```

Comment:
This test exhibit a more general issue with `SemanticCheckerImpl::checkComplexRule`, as commented by `b-scholtz` in the function:
```
    // Count the variable occurrence for the body of a
    // complex rule only once.
    // TODO (b-scholz): for negation / disjunction this is not quite
    // right; we would need more semantic information here.
```